source/info-tool/ev3dev-system-info.sh

```diff
@@ -32,6 +36,14 @@ print_val() {
   printf "%-20s%s\n" "$1: " "$2"
 }
 
+print_fence_if_markdown() {
+  if [ "$FORMAT_MARKDOWN" = true ]; then
```

Comment:
I probably wouldn't have said anything, but it jumped out at me because `true` is also a command. The quotes make it obvious that it is a string here.